BenadrylAutomation/src/test/java/com/jnj/testcases/FooterPageTest.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * @author Rashi Tiwari
+ * @date 18-March-24
+ */
+package com.jnj.testcases;
+
+import org.testng.SkipException;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import com.jnj.base.BaseClass;
+import com.jnj.dataproviders.FooterProvider;
+
+public class FooterPageTest extends BaseClass {
+	
+	@BeforeMethod
+	public void setup() throws InterruptedException {
+		launchApplication();
+	}
+	
+	@Test(priority = 1, description = "Verify the footer link of first column", dataProvider = "footer", dataProviderClass = FooterProvider.class, enabled = true)
+	public void Footer_verifyFirstColumn(String testcase, String execution, String link , String expectedUrl) {
+		test = test.createNode(testcase);
+		if (execution.equalsIgnoreCase(defaultFlag)) {
+			footer.verifyFooterlink(link, expectedUrl);
+		} else {
+			throw new SkipException("Test skipped : " + testcase);
+		}
+	}  
+	
+	@Test(priority = 2, description = "Verify the footer link of second column", dataProvider = "footerSecondColumn", dataProviderClass = FooterProvider.class, enabled = true)
+	public void Footer_verifySecondColumn(String testcase, String execution, String link , String expectedUrl) {
+		test = test.createNode(testcase);
+		if (execution.equalsIgnoreCase(defaultFlag)) {
+			footer.getSecondfooteroption(link, expectedUrl);
```

Comment:
@RashiTiwari21 please put the below two lines first thing in all tests inside the if loop

if (execution.equalsIgnoreCase(defaultFlag)) {
**selectEnv(runOn);
home.closePrivacyPopup();**
// after this your code will come
}

Reply:
fixed

Reply:
Verified @RashiTiwari21